lib/csv_query/database.rb

```diff
@@ -1,37 +1,62 @@
 require 'sqlite3'
+require 'csv_query/table'
 
 module CsvQuery
-  # Wraps a SQLite in-memory database with a single table named csv.
+  # Wraps a SQLite in-memory database with a single table named csv by default if you pass in only a csv to initialize.
+  # Don't pass in a CSV and use #create_table and #import_data_from_csv to to perform more complex multi-table loads.
   class Database
-    attr_reader :database
+    attr_accessor :sqlite_database
```

Comment:
Good point. `database.sqlite_database` seems a bit much, though. I'd be ok with `#sqlite`, I think. How about `#backend`?